mkosi/__init__.py

```diff
@@ -2174,9 +2196,9 @@ def generate_secure_boot_key(args: MkosiArgs) -> None:
         "-new",
         "-x509",
         "-newkey", f"rsa:{keylength}",
-        "-keyout", "mkosi.secure-boot.key",
-        "-out", "mkosi.secure-boot.crt",
-        "-days", str(args.secure_boot_valid_days),
+        "-keyout", "mkosi.key",
+        "-out", "mkosi.crt",
+        "-days", str(args.valid_days),
         "-subj", f"/CN={cn}/",
```

Comment:
Not sure whether we *should* recycle keys for other than secureboot stuff, but I guess it's fine until somebody complains. I would prefer a less generic name than `args.valid_days`, though, e.g. `args.keypar_valid_days`. Same for the common name.